docs/source/reference/kubernetes/kubernetes-setup.rst

```diff
@@ -243,6 +244,15 @@ To simplify the setup, we provide a `script <https://github.com/skypilot-org/sky
 
 You may distribute the generated kubeconfig file to users who can then use it to submit tasks to the cluster.
 
+
+.. _kubernetes-setup-priority:
+[Optional] Step 5 - Set up priority and preemption
```

Comment:
This is for now added as step 5, to be refactored in https://github.com/skypilot-org/skypilot/pull/4934